src/site/resources/css/site.css

```diff
@@ -164,7 +164,7 @@ a[title="toTop"] {
     text-align: left;
   }
 
-  section[id$="Properties"] .wrapper .bodyTable tr td:first-child {
+  section[id="Properties"] .wrapper .bodyTable tr td:first-child {
```

Comment:
"ends with" -> "equals"

https://www.w3schools.com/cssref/sel_attr_end.php

Reply:
> Why are some checks present in the sidebar, but not in the RHS check type summary?

This specific example of `Annotations` was the first one we merged. I remember I did the table in `index.html` manually so probably that's why. The other categories were automated.

I opened an issue at https://github.com/checkstyle/checkstyle/issues/13301 for the test. I will try to write it today.

Fixing this right now.

Reply:
Done.